### PR TITLE
Add readme, contributing and license

### PR DESCRIPTION
**Pull request by gugutz**
- add basic package description
- add install instructions
- add usage instructions on telegram
- add contributing section
- checked 'Write Readme' on Todo list
- add license

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,71 @@
...
+# Table of Contents
+
+-   [Install](#install)
+-   [Githooks](#githooks)
+-   [Usage](#usage)
+-   [Contributing](#contributing)
+-   [License](#license)
+
+
+## Installation
```
Comment (thiamsantos) [inline README.md]: Could you move this to a contributing file? Maybe use [this file](https://github.com/thiamsantos/red_mutex/blob/master/CONTRIBUTING.md) as base and add a section on how to run locally.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,71 @@
...
+## Usage
+
+To use the bot, once its running, invoke it from any chat on Telegram using the inline syntax:
+`@bot_name <name-of-melpa-package>`
+
+Example:
+
+![info box screenshot](bot-usage-example.png "Package info screenshot")
+
+## Githooks
```
Comment (thiamsantos) [inline README.md]: Also, the githooks section should be on the contributing file, would you care to move it?

File: README.md [hunk 3]
```diff
@@ -16,12 +75,10 @@
 - [x] Configure distillery
 - [x] Deploy on gigalixir
 
-## Githooks
+## License
+
+[MIT license](LICENSE) &copy; [Thiago Santos](https://github.com/thiamsantos)
```
Comment (thiamsantos) [inline README.md]:
```suggestion
[MIT License](LICENSE) &copy; [Thiago Santos](https://github.com/thiamsantos)
```
Could you add a `LICENSE` file to the repo?

File: README.md [hunk 4]
```diff
@@ -1,12 +1,71 @@
...
+To install the bot locally:
+
+- Clone the repo: `git clone https://github.com/thiamsantos/melpa_telegram_bot`
+- Install the [githooks](#githooks)
+- Navigate to the newly cloned directory: `cd melpa_telegram_bot`
+- Run `mix deps.get`
+- Run `docker-compose up` to mount the database instance
+- Run `mix run --no-halt` to run the bot
+
+## Usage
```
Comment (thiamsantos) [inline README.md]: Maybe we could remove this usage section and just add an gif to the initial description. WDYT?

File: README.md [hunk 5]
```diff
@@ -1,12 +1,71 @@
...
+# Melpa Telegram Bot
 
 [![Build Status](https://travis-ci.com/thiamsantos/melpa_telegram_bot.svg?branch=master)](https://travis-ci.com/thiamsantos/melpa_telegram_bot)
 [![Coverage Status](https://coveralls.io/repos/github/thiamsantos/melpa_telegram_bot/badge.svg?branch=master)](https://coveralls.io/github/thiamsantos/melpa_telegram_bot?branch=master)
 
+This is a Telegram bot that fetches information from [Melpa Emacs Package Repository](melpa.org) and displays it on a nice card-style format.
+
+![info box screenshot](bot-screenshot.png "Package info screenshot")
+
+Made with Elixir and the [Nadia](https://github.com/zhyu/nadia) telegram bot API wrapper.
```
[the remark this answers is not in the document]
Comment (gugutz) [reply README.md]: sure. removed

File: README.md [hunk 6]
```diff
@@ -1,12 +1,71 @@
-# MelpaBot
+# Melpa Telegram Bot
 
 [![Build Status](https://travis-ci.com/thiamsantos/melpa_telegram_bot.svg?branch=master)](https://travis-ci.com/thiamsantos/melpa_telegram_bot)
 [![Coverage Status](https://coveralls.io/repos/github/thiamsantos/melpa_telegram_bot/badge.svg?branch=master)](https://coveralls.io/github/thiamsantos/melpa_telegram_bot?branch=master)
 
+This is a Telegram bot that fetches information from [Melpa Emacs Package Repository](melpa.org) and displays it on a nice card-style format.
```
[the remark this answers is not in the document]
Comment (gugutz) [reply README.md]: Corrected this one

File: README.md [hunk 7]
```diff
@@ -1,12 +1,71 @@
...
+
+![info box screenshot](bot-screenshot.png "Package info screenshot")
+
+Made with Elixir and the [Nadia](https://github.com/zhyu/nadia) telegram bot API wrapper.
+
+It works by fetching `melpa.org/archive.json` and `melpa.org/download_counts.json`, parsing them and then storing the package information on a database every 1 hour. The database is then used to get the information about the package requested.
```
[the remark this answers is not in the document]
Comment (gugutz) [reply README.md]: Done!

File: README.md [hunk 8]
```diff
@@ -1,12 +1,71 @@
...
+
+# Table of Contents
```
[the remark this answers is not in the document]
Comment (gugutz) [reply README.md]: Sure. Section removed.

File: README.md (hunk 2 above)
Comment (gugutz) [reply README.md]: ofc. moved it.

File: README.md (hunk 1 above)
Comment (gugutz) [reply README.md]: alright. used the linked file as a base as suggested and made proper adjustments

File: README.md (hunk 3 above)
Comment (gugutz) [reply README.md]: added LICENSE file and changed to Apache license.

File: README.md (hunk 4 above)
Comment (gugutz) [reply README.md]: i think its a great idea. gifs are very ilustrative, and great for this cases (usage examples)

made the gif and added to the package description.

File: README.md [hunk 9]
```diff
@@ -1,12 +1,18 @@
-# MelpaBot
+# Melpa Telegram Bot
 
 [![Build Status](https://travis-ci.com/thiamsantos/melpa_telegram_bot.svg?branch=master)](https://travis-ci.com/thiamsantos/melpa_telegram_bot)
 [![Coverage Status](https://coveralls.io/repos/github/thiamsantos/melpa_telegram_bot/badge.svg?branch=master)](https://coveralls.io/github/thiamsantos/melpa_telegram_bot?branch=master)
 
+An inline bot for Telegram that fetches information from [Melpa Emacs Package Repository](melpa.org) and displays it on a nice card-style format.
+
+![bot usage demostration](usage.gif "Usage demostration")
```
Comment (thiamsantos) [inline README.md]:
```suggestion
![bot usage demonstration](usage.gif "Usage demonstration")
```

File: README.md [hunk 10]
```diff
@@ -1,12 +1,18 @@
...
+
+It works by fetching https://melpa.org/archive.json and https://melpa.org/download_counts.json, parsing them and then storing the package information on a database every 1 hour. The database is then used to get the information about the package requested.
+
 ## TODO
```
Comment (thiamsantos) [inline README.md]: Could you remove the TODO section. I created the appropriate issues on the repository itself.

File: README.md [hunk 11]
```diff
@@ -16,12 +22,14 @@
...
 
-## Githooks
+## Contributing
+
+See the [contributing file](CONTRIBUTING.md).
+
+## License
+
+[Apache License, Version 2.0](LICENSE) © [Thiago Santos](https://github.com/thiamsantos)
+
```
Comment (thiamsantos) [inline README.md]: Could you removed these extra lines?

File: README.md (hunk 9 above)
Comment (gugutz) [reply README.md]: fixed this typo

File: README.md (hunk 10 above)
Comment (gugutz) [reply README.md]: Cool. Removed it then.

File: README.md (hunk 11 above)
Comment (gugutz) [reply README.md]: done. i left only 1 extra line, as i recall its recommended. is that right?

**Pull request closed without merging**